pkg/sidecar/bundle_blob.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Doesn't look like this is used yet. Remove?

Reply:
Yeah that's fair - I had it in as a guide for what the JSON should look like but there's no need for it right now. I'll remove!